### ISSUE TITLE
Feature Request: WCS locking between different image displays in the same notebook

### ISSUE THREAD

**Issue by pllim** [astropy/astrowidgets#62]
~~Currently, this works for "pixel locking"~~, but what about "WCS locking"; i.e., ImageWidget A and ImageWidget B (both displayed in the same notebook) have similar pointing but different central RA and DEC and/or rotation, what does it mean to lock their WCS when one pan/zoom on one of them? How is it accomplished? Do we need to rotate one of them so they both have the same orientation?
```python
import ipywidgets

mylink = ipywidgets.widgets.jslink((w._jup_img, 'value'), (w2._jup_img, 'value'))
```

UPDATE: `jslink` value does not really do what I want...

xref spacetelescope/dat_pyinthesky#7

**Comment by pllim** [comment]
Revisiting this again. Use cases:

* 1 window to "blink" 2 candidates manually with a "hot key"
* 2 linked windows with "a hundred" candidates to load for each
* 12 linked windows with one candidate loaded for each

Need to support GWCS. Ignore reprojection (drizzle) for now (ejeschke/ginga#328).

cc @eteq , @hcferguson, and @larrybradley

**Comment by pllim** [comment]
All 3 cases implemented in #106 . Please review them.

**Comment by pllim** [comment]
We ended up going with glue-jupyter that support this with glue linking outside of astrowidgets.
